config/application.rb

```diff
@@ -26,6 +26,11 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     # Don't generate system test files.
-    config.generators.system_tests = nil
+    config.generators do |g|
+      g.system_tests false
+      g.assets false
+      g.helper false
+      g.view false
```

Comment:
Do we want to disable view generators? I can see us using those for the admin UI.

Reply:
We don't think we'll need the view generators for the near future, given we'll mostly be working on the APIs. We can add views manually, or add this generator back when we need it.

Reply:
I'm ok with that.